### DISCUSSION TITLE
As Global Admin that need grant permissions to other member to run "Invoke-Maester", What is the minimum role to grant?

### DISCUSSION THREAD

**Discussion by sergioandreslq** [maester365/maester#442] [Q&A]
Hello team,

If I am MSP that manage customer's tenants, Let's suppose I will do a security review of third-party tenant, what roles should I request to the GA to be able to perform the Invoke-Maester?

If I am the GA of the tenant and need to grant permission to other member to run: "Invoke-Maester" for the test: CISA-EIDSCA-MAESTER.

Which role I need to grant to that member additional to:

    View-Only Configuration OR
    O365SupportViewConfig

![image](https://github.com/user-attachments/assets/478d9022-e47f-482e-b286-4a42f64ddfd6)

**Comment by soulemike** [comment]
There isn't a direct correlation between the Entra built-in roles and the more granular permissions. Global Reader for the tenant would likely give the correct access. You'd likely be better suited provisioning an App Registration with the granular permissions and allowing that individual to use the authentication of the service principal.
